pkg/service/job_service.go

```diff
@@ -0,0 +1,31 @@
+package service
+
+import (
+	"github.com/b-open/jobbuzz/pkg/model"
+)
+
+func (s *Service) GetJobs() ([]model.Job, error) {
+	var jobs []model.Job
+
+	results := s.Database.Find(&jobs)
+
+	if err := results.Error; err != nil {
+		return nil, err
+	}
+
+	if results.RowsAffected < 1 {
+		return make([]model.Job, 0), nil
+	}
```

Comment:
I don't think this is correct? I am not sure whether RowsAffected works for select queries.

What I was thinking is that the default sql package return `ErrNoRows` error if the select query returns empty, and if we don't want that to run into an error, we will need to check for it and return nil or empty slice instead.

It seems that gorm probably doesn't return any error if the rows are empty, except for certain functions. So we can safely remove line 16 to 18.

https://gorm.io/docs/error_handling.html#ErrRecordNotFound

Reply:
Seems like previously Find did return an error, but starting for v1.20 it no longer does.